app/views/audits/index.html.haml

```diff
@@ -18,7 +18,7 @@
           - if audit.action == 'destroy'
             %td #{audit.auditable_type} (#{audit.auditable_id})
           - else
-            %td= audit_model_link(audit, audit.auditable)
+            %td= audit_model_link(audit, safe_auditable(audit))
```

Comment:
Here's what this will look like when the model is missing:

<img width="784" alt="Screenshot 2025-02-05 at 8 25 02 AM" src="https://github.com/user-attachments/assets/c365c10e-9b01-44ae-87dc-c34df4862053" />